src/expr/src/scalar/mod.rs

```diff
@@ -77,28 +78,147 @@ pub enum MirScalarExpr {
 }
 
 impl Arbitrary for MirScalarExpr {
-    // TODO: This is just a stub. #11740
     type Parameters = ();
-    type Strategy = Just<MirScalarExpr>;
+    type Strategy = BoxedStrategy<MirScalarExpr>;
 
     fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
-        prop_oneof![Just(MirScalarExpr::Column(0))]
```

Comment:
Can you add add a TODO comment here?

Reply:
I filled in the entire implementation for MirScalarExpr, so that is no longer necessary.